src/mongo/db/commands/feature_compatibility_version.cpp

```diff
@@ -207,8 +207,6 @@ void FeatureCompatibilityVersion::set(OperationContext* txn, StringData version)
         // featureCompatibilityVersion to 3.4. This is a new index version that isn't supported by
         // versions of MongoDB earlier than 3.4 that will cause 3.2 secondaries to crash when it is
         // replicated.
-        std::vector<BSONObj> indexSpecs{k32IncompatibleIndexSpec};
```

Comment:
I'm new to github. How can I ping @visemet? I've always been convinced that if a local variable is declared but not used in appropriate scope, it cannot influence program behaviour. Am I wrong?